src/hotspot/os_cpu/linux_aarch64/os_linux_aarch64.cpp

```diff
@@ -140,6 +141,8 @@ frame os::fetch_compiled_frame_from_context(const void* ucVoid) {
   intptr_t* sp = os::Linux::ucontext_get_sp(uc);
   address pc = (address)(uc->uc_mcontext.regs[REG_LR]
                          - NativeInstruction::instruction_size);
+  // Compiled code should not have signed the return address.
+  assert(pauth_ptr_is_raw(pc), "cannot be signed");
```

Comment:
Do we need this, now that we have the assert in frame constructor ?

Reply:
Removed as suggested.

Also fixed some build failures.